.scalafmt.conf

```diff
@@ -7,7 +7,6 @@ maxColumn = 100
 align = none
 
 rewrite.rules = [
-  AvoidInfix
```

Comment:
Why are we removing this?

Reply:
it changes all expressions like:
`something.map(_ or _)`
into
`something.map(_.or(_))`

Reply:
same thing for our unit tests

Reply:
Yes, `.map(_.or(_))` is better, and should probably be written as `.map((x, y) => x.or(y))` anyway. Formatting in unit tests is less strict (for now) and should probably be visited to be rewritten in a style different than FlatSpec some day.

Reply:
I put it back